### ISSUE TITLE
Player control mappings don't work

### ISSUE THREAD

**Issue by kflak** [martineausimon/nvim-lilypond-suite#5]
Hi,

I like the look of the new player! One thing, though: I can only get the keys `p` `q` and `l` to work for respectively play, quit and loop. I copied the setup from the readme and tweaked it a little bit:
```
require('nvls').setup({
    lilypond = {
        mappings = {
            player = "<F3>",
            compile = "<F5>",
            open_pdf = "<F6>",
            switch_buffers = "<C-Space>",
            insert_version = "<F4>"
        },
        options = {
            pitches_language = "english",
            output = "pdf"
        },
    },
    latex = {
        mappings = {
            compile = "<F5>",
            open_pdf = "<F6>",
            lilypond_syntax = "<F3>"
        },
        options = {
            clean_logs = false
        },
    },
    player = {
        mappings = {
            quit = "q",
            play_pause = "p",
            loop = "<A-l>",
            backward = "h",
            small_backward = "<S-h>",
            forward = "l",
            small_forward = "<S-l>",
            decrease_speed = "j",
            increase_speed = "k",
            halve_speed = "<S-j>",
            double_speed = "<S-k>"
        },
        options = {
            row = "2%",
            col = "99%",
            width = "37",
            height = "1",
            border_style = "single",
            winhighlight = "Normal:Normal,FloatBorder:Normal"
        },
    },
})

vim.cmd([[autocmd FileType lilypond set formatoptions-=cro]])

```
I'm on Arch linux, Nvim 0.8.0, and all the dependencies are installed and working.

**Comment by martineausimon** [comment]
Hi !

I've just copy-paste your config and it works for me... It looks like you enter player in insert mode. 
Do you have a plugin or a mapping witch does this behavior ?

**Comment by martineausimon** [comment]
I've just added `<cmd>stopinsert<cr>` before each mappings, to prevent insert mode. Let me know if it works !

**Comment by kflak** [comment]
Thanks for the quick response!

I tried with a minimal init.lua, and it works as advertised. However, when running it from my normal (possibly too bloated) init file, the player still opens in insert mode, even with the changes you made.

OK, I located the problem: I have this in my autocommands:
```
api.nvim_create_autocmd({"TermOpen"}, {
     callback = function() require'settings'.term_settings() end,
     group = group
 })
```
calling this function:
```
function M.term_settings()
    vim.api.nvim_command("startinsert")
end
```
When I disable the autocmd the player works as advertised. So the problem seems to be that the autocmd is triggered _after_ your stopinsert still. Don't really know how to work around that. For the time being I am fine with starting my terminals in normal mode, though.
